src/Engines/thinEngine.ts

```diff
@@ -428,6 +428,11 @@ export class ThinEngine {
      */
     public disableVertexArrayObjects = false;
 
+    /**
+     * Indicates that this engine supports render passes
+     */
+    public readonly supportRenderPasses = false;

```

Comment:
Maybe you only have a public get {} and a protected value ?

Reply:
Also does it need to be on ThinEngine or Engine ???

Reply:
I need it in `ThinEngine` because in some code (I think the `ParticleSystem`) we can be passed a `ThinEngine` and I need to know if the engine supports render passes or not.